build/choco-theme.ts

```diff
@@ -207,13 +207,13 @@ const init = async () => {
                     source: `${repositoryConfig.theme.root}.eslintrc.js`,
                     destination: `${repository.root}.eslintrc.js`,
                     isFolder: false
-                },
-                {
+                }
+                /*{
                     task: 'tsconfig.json',
                     source: `${repositoryConfig.theme.root}tsconfig.json`,
                     destination: `${repository.root}tsconfig.json`,
                     isFolder: false
-                }
+                }*/
```

Comment:
I have amended my commit to remove this chunk entirely instead of commenting it out. 

It turns out that with the recent changes to choco-theme, the configuration for TypeScript stuff is included with the esbuild configuration in choco-theme. This means we do not need this file in each repository anymore. 

I have tested removing this file from the portal, adding a TypeScript file, and then rebuilding, and it compiles the TS perfectly without this config file present. I have also tested the Playwright test files on chocolatey.org with the same approach. 

I originally removed it because it was causing strange build errors in Visual Studio, and then discovered it's not even needed.

cc @corbob